src/huggingface_hub/commands/user.py

```diff
@@ -95,12 +139,51 @@ def __init__(self, args):
 
 class LoginCommand(BaseUserCommand):
     def run(self):
-        login(token=self.args.token, add_to_git_credential=self.args.add_to_git_credential)
+        login(
+            token=self.args.token,
+            profile_name=self.args.profile_name,
+            add_to_git_credential=self.args.add_to_git_credential,
+        )
 
 
 class LogoutCommand(BaseUserCommand):
     def run(self):
-        logout()
+        logout(profile_name=self.args.profile_name)
+
+
+class AuthSwitchCommand(BaseUserCommand):
+    def run(self):
+        profile_name = self.args.profile_name
+        if profile_name is None and not self.args.disable_tui:
+            profile_name = self._select_profile_tui()
```

Comment:
Sorry for suggesting the TUI option. Maybe for a first PR that was not the best idea as it can lead to more friction :see_no_evil: What I would suggest here to avoid friction:
- remove `--disable-tui` option
- if InquirerPy installed => `self._select_profile_tui()`
- if not installed => a simple "print a numbered list" + "type a number" UI to select it

Therefore, `InquirerPy` is not a requirement anymore and not having it still leads to a good-enough interface.